flow-typed/forms.js

```diff
@@ -422,6 +422,21 @@ declare type FormPostSubmissionAction =
   | 'CLOSE'
   | 'FORMS_LIBRARY'
 
+type FormValidationEndpoint =
+  | {
+      type: 'CALLBACK',
```

Comment:
```suggestion
      type: 'URL',
```

Reply:
This is to make consistent with submission events